pvtrace/Trace.py

```diff
@@ -724,8 +761,12 @@ def container(self, photon):
             return containers[min_index]
 
 class Tracer(object):
```

Comment:
I would call this class `ParallelTracer` so that the namespace does not have too much duplication,